byteps/common/core_loops.cc

```diff
@@ -13,14 +13,15 @@
 // limitations under the License.
 // =============================================================================
 
+#include "core_loops.h"
```

Comment:
could you please move this header file to its original position?

Reply:
um, ok. My linter did it automatically.